### PR TITLE
improvement: Add connection error messaging for bitflyer

### PR DESCRIPTION
**Pull request by OAuthBringer**
Pretty much rinse and repeat.

### CODE REVIEW

File: app/javascript/views/walletServices/braveConnection/BitflyerConnection.tsx [hunk 1]
```diff
@@ -85,9 +100,9 @@ class BitflyerConnection extends React.Component<any, any> {
                         />
                     </div>
                 </div>
-                {!this.props.isPayable && (
-                    <VerifyButton verifyUrl={this.props.verifyUrl}>
-                        <FormattedMessage id="walletServices.bitflyer.notPayable" />
+                {hasProblem && (
+                    <VerifyButton verifyUrl={verifyUrl}>
```
Comment (jlbyrne) [inline app/javascript/views/walletServices/braveConnection/BitflyerConnection.tsx:104]: Did we want to fix the verify button issue here, or should that be a separate issue?

Comment (OAuthBringer) [reply app/javascript/views/walletServices/braveConnection/BitflyerConnection.tsx:104]: I can fix it really quickly

Comment (OAuthBringer) [reply app/javascript/views/walletServices/braveConnection/BitflyerConnection.tsx:104]: @jlbyrne patched

**Review by jlbyrne — APPROVED** [review]
LGTM!

**Pull request merged**